redaxo/src/addons/debug/install.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+$addon = rex_addon::get('debug');
+rex_dir::copy(
+    $addon->getPath('vendor/itsgoingd/clockwork/Clockwork/Web/public'),
+    $addon->getAssetsPath('clockwork')
```

Comment:
die index.html, die wir hier kopieren, ist dann direkt aufrufbar, selbst wenn debug-modus aus und debug-addon sogar deaktiviert, aber noch installiert.
Ist das ein Problem, oder kann man da nix mit anfangen, selbst wenn man sie aufruft?

Reply:
das ist das selbe frontend, dass auch für die browser extension verwendet wird. also egal, ob mann es aufrufen kann.
Hier ist es wichtiger die rex-api von dem debug addon, dass die daten liefert zu "schützen". also erstmal an den debug-mode knüpfen und ich schau mal in wiefern der mit die backend-session mitnutzen kann (für webapge vermutlich kein problem, aber in extension).

Reply:
aktuell ist es an den Debug-Mode geknüpft und an eine Backend-Admin-Session.

es macht aktuell den Zugang über die Extension kaputt, da schau ich mal wie gut die Authentifizierung Dinger von Clockwork funktionieren.

Reply:
habe nochmal drüber nachgedacht und bin mir unsicher, ob man nicht nur das als webapp und nicht über die extensions anbieten sollte.
Durch die Webapp hat man immer den passenden Stand frontend + backend. Die extensions können teilweise neuer und evt. auch inkompatibel werden (da ja selbst major versionen vom Browser automatisch installiert werden).